src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -797,6 +805,12 @@ static constexpr Property<bool, PropertyMutability::RW> force_tbb_terminate{"FOR
  */
 static constexpr Property<bool, PropertyMutability::RW> enable_mmap{"ENABLE_MMAP"};
 
+/**
+ * @brief Read-write property to set the percentage of the model can be queried during query model
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+static constexpr Property<float, PropertyMutability::RW> query_model_ratio{"QUERY_MODEL_RATIO"};
+
```

Comment:
about this property, seems should be an internal property, or you want to let user manipulate this ratio?

Reply:
It's an internal property now.